AdyenDropIn/Presentation/WrapperViewController.swift

```diff
@@ -37,14 +37,18 @@ internal final class WrapperViewController: UIViewController {
 
         return viewController?.children.contains(where: { heirarchyRequiresKeyboardInput(viewController: $0) }) ?? false
     }
+    
+    private var frameUpdateThrottler = Throttler(minimumDelay: 0.05)
 
     internal func updateFrame(keyboardRect: CGRect) {
         guard let view = child.viewIfLoaded else { return }
         let finalFrame = child.finalPresentationFrame(with: keyboardRect)
         topConstraint?.constant = finalFrame.origin.y
         leftConstraint?.constant = finalFrame.origin.x
         rightConstraint?.constant = -finalFrame.origin.x
-        view.layoutIfNeeded()
+        frameUpdateThrottler.throttle {
```

Comment:
possible to write `frameUpdateThrottler.throttle(view.layoutIfNeeded)`, not sure if better :)

Reply:
👍